docs/tables/aws_route53_vpc_association_authorization.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Steampipe Table: aws_route53_vpc_association_authorization - Query AWS Route53 for other-account VPCs."
+description: "Gets a list of the VPCs that were created by other accounts and that can be associated with a specified hosted zone because you've submitted one or more `CreateVPCAssociationAuthorization` requests.."
```

Comment:
```suggestion
description: "Gets a list of the VPCs that were created by other accounts and that can be associated with a specified hosted zone because you've submitted one or more `CreateVPCAssociationAuthorization` requests."
```